File: src/Programmers/P1/P1_12954.java
package Programmers.P1;

import java.util.Arrays;
import java.util.stream.Collectors;

public class P1_12954 {
    public static void main(String[] args) {
        long[] solution = solution(1, 1);
        for (long l : solution)
            System.out.println(l);
    }

    public static long[] solution(int x, int n) {
        long[] answer = new long[n];
        int i = 0;
        while(i<n){
            answer[i] = x*(1+i++);
        }
        return answer;
    }
}

File: src/Programmers/P1/P1_12954.kt
package Programmers

fun main() {
    println(solution(2, 5).contentToString())
    println(solution(4, 3).contentToString())
    println(solution(-4, 2).contentToString())
}

private fun solution(x: Int, n: Int): LongArray {
    return (1L..n).map { v -> v * x }.toLongArray()
}